src/app/modules/sgid/sgid.service.ts

```diff
@@ -144,7 +178,7 @@ export class SgidService {
 
   /**
    * Verifies a sgID JWT and extracts its payload.
-   * @param jwtSgid The contents of the JWT cookie
+   * @param {String} jwtSgid The contents of the JWT cookie
    */
   extractJWTInfo(
```

Comment:
nit: here too, change name to `createJwtInfo` instead?

Reply:
We're not creating anything here; we are extracting information out of a JWT